about.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "Discourse Header Search",
+  "name": "Header Search",
   "component": true,
-  "about_url": null,
+  "about_url": "https://meta.discourse.org/t/header-search/194093",
   "license_url": null
```

Comment:
We should add a license file 😛

Reply:
Yeah, I thought it was weird that a few that are missing a license. I was going to round them up and ask about those once I'd done this burst of updates.